arviz/plots/parallelplot.py

```diff
@@ -69,9 +94,23 @@ def plot_parallel(
     var_names, _posterior = xarray_to_ndarray(
         get_coords(posterior_data, coords), var_names=var_names, combined=True
     )
-
     if len(var_names) < 2:
         raise ValueError("This plot needs at least two variables")
+    if norm_method is not None:
+        if norm_method == "normal":
+            mean = np.mean(_posterior, axis=1)
+            standard_deviation = np.std(_posterior, axis=1)
+            for i in range(0, np.shape(mean)[0]):
```

Comment:
Maybe I am not thinking of the right example, but is this block equivalent to 
```python
_posterior = (_posterior - _posterior.mean(axis=1)) / np.std(_posterior, axis=1)
```

Reply:
Had to perform each operation using loops as the above method threw broadcasting issues.

Reply:
_posterior = (_posterior - _posterior.mean(axis=1, keepdims=True)) / np.std(_posterior, axis=1, keepdims=True)

What is the shape of the `_posterior`?

Reply:
iirc (2000,2).